doc/10-icinga-template-library.md

```diff
@@ -3304,7 +3304,8 @@ fail2ban_path           | **Optional.** Specify the path to the tw_cli binary, d
 fail2ban_warning        | **Optional.** Specify a warning threshold, default is 1
 fail2ban_critical       | **Optional.** Specify a critical threshold, default is 2
 fail2ban_socket         | **Optional.** Specify a socket path, default is unset
-fail2ban_perfdata       | **Optional.** If set to true, activate the perfdata output, default value for the plugin is set to true.
+fail2ban_perfdata       | **Optional.** If set to true, activate the perfdata output, default value for the plugin is set to true
+fail2ban_jail           | **Optional.** Specify the name of the specific jail to monitor; default is to omit and thus monitor all jails.
```

Comment:
Why did you remove the period on the one line (right as conforms to the others), but put one at the end of the other line (wrong)?

Reply:
Because it seems linguistically correct: since it's a listing, we end the last one with a period. So I simply moved the period to the last item in the list.